backend/tests/test_upload_token_security.py

```
@@ -0,0 +1,100 @@
+        "selfie": (BytesIO(b"selfie"), "selfie.jpg"),
+    }
+    if token:
+        data["uploadToken"] = token
+    return data
+
+
+@pytest.fixture()
+def app():
+    app = Flask(__name__)
+    app.config["TESTING"] = True
+    app.config["JWT_SECRET_KEY"] = "test-secret"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "JWT_SECRET_KEY"

(S105)